app/code/community/TIG/MyParcel2014/Block/Adminhtml/Widget/Grid/Column/Renderer/ShippingStatus.php

```diff
@@ -99,19 +99,22 @@ public function render(Varien_Object $row)
             $i = 0;
             foreach ($myParcelShipments as $myParcelShipment) {
 
-                if ($i++ == 1)
+                if ($i ++ == 1) {
```

Comment:
```suggestion
                if ($i++ == 1) {
```
Dit moet wel aan elkaar
https://www.php-fig.org/psr/psr-12/#54-for

Reply:
Done